<launch>
  <arg name="robot_id" default="$(optenv ROBOT_ID unnamed_robot)"/>
  
  <group ns="$(arg robot_id)">
    <node pkg="nmpc_nav_control" type="nmpc_nav_control" name="nmpc_nav_control" output="screen">
      <param name="global_frame_id" value="$(arg robot_id)/map"/>
      <param name="base_frame_id" value="$(arg robot_id)/base_footprint"/>

      <param name="control_freq"                   value="40"/>
      <param name="transform_timeout"              value="0.1"/>
      <param name="max_active_path_length"         value="5.0"/>
      <param name="final_position_error"           value="0.001"/>
      <param name="final_orientation_error"        value="1.0"/>

      <param name="steering_geometry" value="omni4"/>
      <param name="rob_dist_between_front_back_wh" value="0.265"/>
      <param name="rob_dist_between_left_right_wh" value="0.270"/>
    </node>
  </group>
</launch>
